tools/cravatp_score_and_annotate/cravatp_submit.xml

```diff
@@ -0,0 +1,302 @@
+          <option value="Rectum">Rectum</option>
+          <option value="Skin">Skin</option>
+          <option value="Stomach">Stomach</option>
+          <option value="Thyroid">Thyroid</option>
+          <option value="Uterus">Uterus</option>
+        </param>
+      </when>
+      <when value="CHASM+VEST">
+        <param format="tabular" name="classifier" type="select" label="CHASM Classifier">
+          <option value="Bladder">Bladder</option>
+          <option value="Blood-Lymphocyte">Blood-Lymphocyte</option>
+          <option value="Blood-Myeloid">Blood-Myeloid</option>
+          <option value="Brain-Glioblastoma-Multiforme">Brain-Glioblastoma-Multiforme</option>
+          <option value="Brain-Lower-Grade-Glioma">Brain-Lower-Grade-Glioma</option>
+          <option value="Breast">Breast</option>
+          <option value="Cervix">Cervix</option>
+          <option value="Colon">Colon</option>
+          <option value="GID">GID</option>
+          <option value="Head-and-Neck">Head-and-Neck</option>
+          <option value="Kidney-Chromophobe">Kidney-Chromophobe</option>
+          <option value="Kidney-Clear-Cell">Kidney-Clear-Cell</option>
+          <option value="Kidney-Papillary-Cell">Kidney-Papillary-Cell</option>
+          <option value="Liver-Nonviral">Liver-Nonviral</option>
+          <option value="Liver-Viral">Liver-Viral</option>
+          <option value="Lung-Adenocarcinoma">Lung-Adenocarcinoma</option>
+          <option value="Lung-Squamous-Cell">Lung-Squamous-Cell</option>
+          <option value="Other">Other</option>
+          <option value="Ovary">Ovary</option>
+          <option value="Pancreas">Pancreas</option>
+          <option value="Prostate-Adenocarcinoma">Prostate-Adenocarcinoma</option>
+          <option value="Rectum">Rectum</option>
+          <option value="Skin">Skin</option>
+          <option value="Stomach">Stomach</option>
+          <option value="Thyroid">Thyroid</option>
+          <option value="Uterus">Uterus</option>
+        </param>
+      </when>
+    </conditional>
+    <param format="tabular" name="GRCh" type="select" label="Genome Reference Consortium Human Build (GRCh)"
+        help="The default human reference genome used for annotation is GRCh38, released on December 24th, 2013 from the Genome Reference Consortium.">
+      <option value="GRCh38">GRCh38/hg38</option>
+      <option value="GRCh37">GRCh37/hg19</option>
+    </param>
+  </inputs>
+  <outputs>
+    <collection name="results" type="list" label="CRAVAT Results on ${on_string}">
+      <data format="tabular" label="CRAVAT: Gene Level Annotation Report on ${on_string}" name="Gene" />
+      <data format="tabular" label="CRAVAT: Variant Report on ${on_string}" name="Variant" />
+      <data format="tabular" label="CRAVAT: Non-coding Variant Report on ${on_string}" name="Noncoding" />
+      <data format="tabular" label="CRAVAT: Errors on ${on_string}" name="Error" />
```

Comment:
I would write all `name`s with small letters. They become variables and usually we stick to small letters.

Reply:
I used uppercase here since the filenames within the collection on Galaxy match with these variable names. Lowercase looks fine too.
![image](https://user-images.githubusercontent.com/32586466/44160708-d71c1500-a080-11e8-89de-d09ecb73b886.png)